torch_xla/csrc/ops/amp_foreach_non_finite_check_and_unscale.cpp

```diff
@@ -0,0 +1,50 @@
+#include "torch_xla/csrc/ops/amp_foreach_non_finite_check_and_unscale.h"
+
+#include "tensorflow/compiler/xla/shape_util.h"
+#include "tensorflow/compiler/xla/xla_client/util.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/xla_ops.h"
+#include "torch_xla/csrc/tensor_util.h"
+#include "torch_xla/csrc/xla_lower_util.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+xla::Shape NodeOutputShape(const OpList& inputs) {
+  std::vector<xla::Shape> output_shapes;
+  output_shapes.reserve(inputs.size() - 1);
+  for (size_t i = 0; i < inputs.size() - 2; ++i) {
+    const xla::Shape& input_shape = inputs[i].shape();
+    output_shapes.push_back(input_shape);
+  }
+  output_shapes.push_back(xla::ShapeUtil::MakeShape(
+      inputs[inputs.size() - 2].shape().element_type(), {}));
+  return xla::ShapeUtil::MakeTupleShape(output_shapes);
+}
+
+}  // namespace
+
+AmpForachNonFiniteCheckAndUnscale::AmpForachNonFiniteCheckAndUnscale(
+    const OpList& inputs)
+    : Node(xla_amp_foreach_non_finite_check_and_unscale, inputs,
```

Comment:
`ir::OpKind(at::aten:: _amp_foreach_non_finite_check_and_unscale_)`not worked? We usually define the OpKind if it is a xla specified OP. Pytorch should have most of the OpKind defined already.

Reply:
No, it didn't work.

Reply:
@ailzhang I think we should add this symbol to pytorch similar to https://github.com/pytorch/pytorch/pull/36851, wdyt?

Reply:
Thanks @JackCaoG for catching this! Yea adding this symbol to the list in the linked PR should fix it. Let us know if it doesn't!

Reply:
@JackCaoG @ailzhang I could add the symbol in the corresponding [pytorch PR](https://github.com/pytorch/pytorch/pull/48570).

Reply:
Done.